lib/manageiq_performance/middlewares/memory.rb

```diff
@@ -0,0 +1,58 @@
+module ManageIQPerformance
+  module Middlewares
+    module Memory
+      private
+
+      def memory_initialize
+        require "objspace" if memory_include_memsize?
```

Comment:
Interesting side note about this.  Seems like on first invocation of this, about 17k-18k objects are instantiated on first invocation.  Which is a bit weird, since it shouldn't be part of the profiled block, but I am not concerned, just pointing it out.

```console
$ bundle exec ruby -e 'require "manageiq_performance/middleware"; ManageIQPerformance.profile("str") { "string" }; ManageIQPerformance.profile("str") { "string" }; ManageIQPerformance.profile("str") { "string" }'
$ cat tmp/manageiq_performance/run_1488819738/str/request_148881973*.memory
---
:total_allocated_objects: 7
:total_freed_objects: 18263
:total_objects: 18270
:old_objects: 0
:memsize_of_all: 2144
---
:total_allocated_objects: 7
:total_freed_objects: 0
:total_objects: 7
:old_objects: 0
:memsize_of_all: 1952
---
:total_allocated_objects: 7
:total_freed_objects: 0
:total_objects: 7
:old_objects: 0
:memsize_of_all: 1952
```